README.md

```diff
@@ -108,14 +106,46 @@ The bundle does add a `certManager` block, that allow to change the version (cur
 
 # Specify the bundle to use
 bundles:
-- targets:
-  - run://quay.io/kairos/community-bundles:cert-manager_latest
+  - targets:
+      - run://quay.io/kairos/community-bundles:cert-manager_latest
 
 # Specify cert-manager settings
 certManager:
   version: v1.11.0
 ```
 
+### Flux
+
+This installs Flux and supports bootstrapping the cluster with [FluxCD](https://fluxcd.io/flux/cmd/flux_bootstrap/). It has logic so only one node will do the bootstrapping,
+by creating a ConfigMap in the `default` namespace named `flux-bootstrap`.
+By default, it will time out after trying for 30 minutes.
+
+```yaml
+#cloud-config
+
+bundles:
+  - targets:
+      # Installs to /usr/local/lib/extensions/flux/
```

Comment:
This is not true. One would have to specify the path (e.g. [like this](https://github.com/kairos-io/kairos/blob/750b0d1d00b029cea57e5913357438b80a85b2c3/tests/assets/live-overlay.tmpl#L9)) and not all paths are available at bundle installation time (I tried various to discover that :D ). 

`run://` bundles on the other hand, they get extracted to a temp directory and the `run.sh` script is called if one is there. This bundle needs to run after kubernetes starts and according to the docs this is how to set it up in the kairos config. I think this needs some playing with `rootfs_path` to get it to extract properly. I see [other bundles installing binaries](https://github.com/kairos-io/community-bundles/blob/4bb2da2de3311023d7ed8c4023fff221c46d9727/coco/run.sh#L33) so that path should be writable at installation time.

To summarize, I think this bundle should:

- install the flux binary using a `run.sh` script (in `/usr/local/bin` for example)
- install a bootstap.sh script in some writable directory using `run.sh` script (not sure what paths are available at that point, I would check what other bundles do).


Then in the kairos config, the bootstrap.sh script should be invoked at the right stage like it happens currently in provided README in this PR. This will ensure the boostrap.sh script will only run when Kubernetes has started.

It needs some trial and error to get this right. Let me know if you need help figuring out a test workflow.

Reply:
Let's leave it as it is now with k3s hardcoded. If someone requests the bundle to work with other k8s distros, we'll figure it out then.